libkbfs/kbfs_ops.go

```diff
@@ -247,6 +246,12 @@ func (fs *KBFSOpsStandard) GetOrCreateRootNode(
 			return nil, EntryInfo{}, err
 		}
 	}
+
+	if !create && md.data.Dir.Type != Dir {
+		kbpki := fs.config.KBPKI()
+		return nil, EntryInfo{}, identifyHandle(ctx, kbpki, kbpki, h)
```

Comment:
Bleh, I forgot about that OS X `readdir` bug.  Ugly.

Ok, well we should add a comment here about the fact that we need to do this to make sure identify errors are returned on open on OS X, since `readdir` doesn't return errors.

Also, `identifyHandle` is fairly expensive and only needs to be run once every 6 hours or so for a given folder.  It's probably worth caching successful identifies in `KBFSOps` (normally they are cached in `folderBranchOps`).